app/models/tag_user.rb

```diff
@@ -11,6 +11,7 @@ class TagUser < ActiveRecord::Base
       .joins("LEFT OUTER JOIN tag_group_permissions ON tag_group_memberships.tag_group_id = tag_group_permissions.tag_group_id")
       .joins("LEFT OUTER JOIN group_users on group_users.user_id = tag_users.user_id")
       .where("(tag_group_permissions.group_id IS NULL
+               OR tag_group_permissions.group_id = 0
                OR tag_group_permissions.group_id = group_users.group_id
```

Comment:
I wonder, does that work? 🤔 

```suggestion
               OR tag_group_permissions.group_id IN  (0, group_users.group_id)
```

Reply:
Looks like does! Thanks!